### DISCUSSION TITLE
uploaded image is not shown

### DISCUSSION THREAD

**Discussion by xizheyin** [django-wiki/django-wiki#1212] [Q&A] [answered]
hi, when I upload a image, it can't be shown.
![image](https://user-images.githubusercontent.com/62123683/190842318-ff44c926-ba17-419e-9571-684654ab366c.png)
![image](https://user-images.githubusercontent.com/62123683/190842342-eac808e2-3235-481a-b2a5-1689f01165d2.png)
 why so? please help me!

**Comment by oscarmcm** [comment, marked as answer]
Hi @changanyyy did you fixed it? This might be related to your media url configuration
